### DISCUSSION TITLE
Facing this error when schedule runs.

### DISCUSSION THREAD

**Discussion by CrocodileDuty** [Taxel/PlexTraktSync#1816] [Support] [answered]
Error: HTTPSConnectionPool(host='api-v2launch.trakt.tv', port=443): Max
         retries exceeded with url: /search/tmdb/149?type=movie (Caused by      
         NameResolutionError("<urllib3.connection.HTTPSConnection object at     
         0xffff80637b50>: Failed to resolve 'api-v2launch.trakt.tv' ([Errno -2] 
         Name or service not known)"))                                          
ERROR    API didn't respond properly, script will abort now. Please try again   
         later.                                                                 
ERROR    Last call: plextraktsync.trakt.TraktApi.search_by_id(('149',),         
         {'id_type': 'tmdb', 'media_type': 'movie'})

**Comment by simonc56** [comment, marked as answer]
read the welcome message please : [Welcome to PlexTraktSync Discussions!](https://github.com/Taxel/PlexTraktSync/discussions/803)

**Reply by CrocodileDuty** [reply]
My apologies!
